js/views/baseVw.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
```

Comment:
This is the aforementioned base view I've talked about, which handles the management of child views (mainly properly removing them to avoid ghost views). I'll explain more in some comments below.